controllers/project_controller_test.go

```diff
@@ -110,6 +114,10 @@ var _ = Describe("ProjectController", func() {
 
 		projectInvalidInput := mockData.CreateProjectInvalidInput
 
+		AfterEach(func() {
+			util.TryDeleteProject(projectInvalidInput.Spec.Name, ns)
```

Comment:
Why are we trying to delete `projectInvalidInput.Spec.Name` in afterEach block ?